datalake/src/main/java/com/sequenceiq/datalake/service/sdx/dr/SdxBackupRestoreService.java

```diff
@@ -339,7 +338,8 @@ private AttemptResult<StackV4Response> getStackResponseAttemptResult(SdxCluster
      */
     private boolean isStackOrClusterStatusFailed(Status status) {
         return Status.BACKUP_FAILED.equals(status) ||
-                Status.RESTORE_FAILED.equals(status);
+                Status.RESTORE_FAILED.equals(status) ||
+                Status.UPDATE_FAILED.equals(status);
```

Comment:
When would we get an update failed status?

Reply:
It is the status for the 'Permission denied' error.
[Errno 13] Permission denied: '/etc/sssd/sssd.conf9x3rp7o_', Name=/etc/sssd/sssd.conf} | One or more requisite failed: sssd.ipa./etc/sssd/sssd.conf